docs/usage/configuration.rst

```diff
@@ -0,0 +1,312 @@
+
+    Send check result of severity :literal:`WARNING` if values arrive apart more than the specified period.
+    This monitors two kinds of failure:
+
+    - The network is fully operational, but two consecutive data points for a metric differ by more than :code:`timeout` in their timestamps.
+      This might indicate that the source for these metrics is not fully operational.
+
+    - :command:`metricq_sink_nsca` does not receive data points for these metrics for more than the specified duration,
+      measured against the local system clock.
+      This might happen if a source has crashed, has lost its connection to the network
+      or there is another issue along the way that prevents clients from consuming new value for these metrics.
+
+    Default
+        Not set; no timeout checks are performed.
+
+    Example
+        Make sure that :literal:`foo.bar` consistently produces values:
+
+        .. code-block:: json
+
+            {
+                "checks": {
+                    "foo": {
+                        "metrics": ["foo.bar"],
+                        "timeout": "1min"
+                    }
+                }
+            }
+
+:code:`ignore` (list of numbers)
+    A list of values that are never considered to generate :literal:`WARNING` or :literal:`CRITICAL` reports.
+
+    This is intended to be used for metrics that yield spurious, but fixed values that should be ignored,
+    even if they are within an otherwise abnormal range.
+    An example would be a faulty measuring device which produces the value :literal:`0.0` on encountering in an internal error,
+    but where :literal:`warning_below = 5.0`.
+
+    .. note::
+
+        **Use with care**.
+        The implementation essentially only performs a floating-point equality test to filter values.
+
+        If this sounds like a bad idea to you, you are probably right.
+        Trust me, this is here because some source cannot be fixed easily.
+
+    Example
+        A source computing a `Power factor <https://en.wikipedia.org/wiki/Power_factor>`_
+        for an AC electrical power system reports a metric :literal:`ac_system.power_factor`.
+        If the power factor is too low, a warning should be generated.
+        It might be that the source calculates a power factor of :literal:`0.0` on low draw.
+        Since a low power factor on low draw might not be considered a problem,
+        ignore the value :literal:`0.0`:
+
+        .. code-block:: json
+
+            {
+                "checks": {
+                    "low-draw": {
+                        "metrics": ["ac_system.power_factor"],
+                        "warning_below": 0.8,
+                        "ignore": [0.0]
+                    }
+                }
+            }
+
+:code:`resend_interval` (:ref:`duration<Duration>`)
+    .. _resend_interval:
+
+    Period of time after which the current state of this check is sent again to the server, eventhough it might not have changed.
```

Comment:
```suggestion
    Period of time after which the current state of this check is sent again to the server, even though it might not have changed.
```